### DISCUSSION TITLE
Office_S_Tips/pki/2019/09/19/Podpisy_cyfrowe

### DISCUSSION THREAD

**Discussion by giscus** [andrzejQ/Office_S_Tips#20] [Komentarze]
# Office_S_Tips/pki/2019/09/19/Podpisy_cyfrowe

Kilka informacji o podpisie cyfrowym i dokumencie elektronicznym.

https://andrzejq.github.io/Office_S_Tips/pki/2019/09/19/Podpisy_cyfrowe.html

**Comment by andrzejQ** [comment]
Nowa wersja komentarzy z użyciem `giscus -> GitHub Discussions`.
